package-lock.json

```diff
@@ -3353,7 +3353,6 @@
         "chalk": "^4.1.2",
         "chokidar": "^3.5.1",
         "cross-spawn": "^7.0.3",
-        "dotenv": "^16.0.0",
```

Comment:
I don't know what happened here but try to not make changes to "package-lock.json" as it causes unnecessary merge conflicts.

Reply:
My mistake. Fixed.